### ISSUE TITLE
`Quaternion.inverse()` is replaced to `Quaternion.invert()` since r123

### ISSUE THREAD

**Issue by FMS-Cat** [yomotsu/camera-controls#151]
### Description

`Quaternion.inverse()` is removed and replaced by `Quaternion.invert()` since r123 (See: https://github.com/mrdoob/three.js/pull/20613)
We uses `Quaternion.inverse()` and we want to remove the use of `inverse()` or Three.js will emit a deprecation warning for each calls of `inverse` in r123.

### Thing I want to make a consensus before I work on this

@yomotsu 
Here's how I resolved this while preserving compatibility with previous versions: https://github.com/pixiv/three-vrm/pull/556/files

```ts
export function quatInvertCompat<T extends THREE.Quaternion>(target: T): T {
  if ((target as any).invert) {
    target.invert();
  } else {
    (target as any).inverse();
  }

  return target;
}
```

Does this look good for you?
If so, I can work on this along with a commit that bumps Three.js to r123

**Comment by yomotsu** [comment]
Thanks for letting me know.
Looks good! I prefer to keep up to date too with your backward compat function.

Would you mind making a PR for it?

**Comment by paulmelnikow** [comment]
I'm running into this warning too, and would love to see this change made!

**Comment by FMS-Cat** [comment]
gonna do this

**Comment by paulmelnikow** [comment]
@FMS-Cat thanks! @yomotsu Could you publish a release as well? 🙏🏼

**Comment by yomotsu** [comment]
@paulmelnikow Absolutely! Just published as 1.25.3✨✨
